File: Java/src/ToLowerCase.java
/**
 * Implement function ToLowerCase() that has a string parameter str, and returns the same string in lowercase.
 * <p>
 * Example 1:
 * <p>
 * Input: "Hello"
 * Output: "hello"
 * Example 2:
 * <p>
 * Input: "here"
 * Output: "here"
 * Example 3:
 * <p>
 * Input: "LOVELY"
 * Output: "lovely"
 * <p>
 * Accepted.
 */

public class ToLowerCase {

    public String toLowerCase(String str) {
        // return str.toLowerCase();
        StringBuilder sb = new StringBuilder();
        for (char ch : str.toCharArray()) {
            if (ch >= 65 && ch <= 90) {
                sb.append((char) (ch + 32));
            } else {
                sb.append(ch);
            }
        }

        return sb.toString();
    }

}

File: Kotlin/src/ToLowerCase.kt
/**
 * Implement function ToLowerCase() that has a string parameter str, and returns the same string in lowercase.
 *
 * Example 1:
 *
 * Input: "Hello"
 * Output: "hello"
 * Example 2:
 *
 * Input: "here"
 * Output: "here"
 * Example 3:
 *
 * Input: "LOVELY"
 * Output: "lovely"
 *
 * Accepted.
 */

class ToLowerCase {

    fun toLowerCase(str: String): String {
//         return str.toLowerCase()
        val sb = StringBuilder()
        for (ch in str.toCharArray()) {
            if (ch.toInt() in 65..90) {
                sb.append((ch.toInt() + 32).toChar())
            } else {
                sb.append(ch)
            }
        }

        return sb.toString()
    }

}
